DPNUnet.imjoy.html

```diff
@@ -446,29 +449,33 @@
     async def predict(self):
         if self.dialog is not None:
             self.dialog.close()
-     
-        ret = await api.showFileDialog(title="Please select a folder containing your testing data ( folder with a subfolder called `test`).", root='/imjoy/imjoy-paper/SEGMENTATION-DATA', type= 'directory', engine= api.ENGINE_URL)
 
+        # Check if model is imported
+        model_path = os.path.join('..', 'weights', 'dpn_softmax_f0', 'fold0_best.pth') # '../weights/dpn_softmax_f0/fold0_best.pth'
+        if not os.path.exists(model_path):
+            api.alert('No model found. Please import model before launching prediction')
+            return
+
+        ret = await api.showFileDialog(title="Please select a folder containing your testing data ( folder with a subfolder called `test`).", root='/imjoy/imjoy-paper/SEGMENTATION-DATA', type= 'directory', engine= api.ENGINE_URL)
         if not ret:
             return
-        elif 'HeLa_DAPI' in ret.path:
-            options_string =  "Average nuclei size { id: 'nuclei_size', type:'number', placeholder: 200}"
-        else:
-            options_string =  "Average nuclei size { id: 'nuclei_size', type:'number', placeholder: 20}"
-        
-        options = await api.showDialog( name="Prediction Configurations", ui= "<br>".join([options_string   ]))
-  
-        config = load_config('configs/imjoy_dpn_softmax_s2.json') 
+        #elif 'HeLa_DAPI' in ret.path:
+        #    options_string =  "Average nuclei size { id: 'nuclei_size', type:'number', placeholder: 200}"
+        #else:
+        #    options_string =  "Average nuclei size { id: 'nuclei_size', type:'number', placeholder: 20}"
+        #options = await api.showDialog( name="Prediction Configurations", ui= "<br>".join([options_string   ]))
+        #scale_factor = 20.0/float(options.nuclei_size)
+        config = load_config('configs/imjoy_dpn_softmax_s2.json')
         config.dataset_path = os.path.join(ret.path, 'test')
         config.results_dir  = os.path.join(ret.path, 'results')
```

Comment:
```suggestion
        config.results_dir  = os.path.join(ret.path, '__outputs__')
```
Currently, there is no actual result in this `results_dir`, so would be better to change it into another name which will be less confusing for the user.

Please find another place for training and change it please.